workflow-dev/tpl/deis-database-deployment.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: deis-database
+  namespace: deis
+  labels:
+    heritage: deis
+  annotations:
+    helm-keep: "true"
+    component.deis.io/version: {{env "DATABASE_GIT_TAG" | default .database.dockerTag}}
+spec:
+  replicas: 1
+  strategy:
+    type: Recreate
```

Comment:
For others to understand, `Recreate` is the important difference here compared to Controller / Router which use `RollingUpdate`